src/main/java/blackjack/domain/card/CardDeck.java

```diff
@@ -0,0 +1,35 @@
+package blackjack.domain.card;
+
+import blackjack.constants.ErrorCode;
+import blackjack.domain.card.exception.NoMoreCardException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CardDeck {
+    private final List<Card> cards;
```

Comment:
전체적으로 한번 통일을 주면 좋을 것 같습니다~

클래스 선언 다음에 개행을 준다. vs 안준다
```suggestion
public class CardDeck {

    private final List<Card> cards;
```